src/qt/transactiondesc.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2011-2021 The Bitcoin Core developers
+// Copyright (c) 2011-2022 The Bitcoin Core developers
```

Comment:
38316d4 Generally there's no need to touch the copyright headers; they are updated annually with a script.

Reply:
Thanks. Done.